### PR TITLE
feat: add keyboard event

### PR DESCRIPTION
**Pull request by hawkingrei**
Hi ~ 
I add keyboard event, you can test it by the following code.

```R
settings <- function() {
    size(500, 500)
}

draw <- function() {
   stdout$print(key)
   #stdout$print(keyCode)
   stdout$print("\n")
}
```

### CODE REVIEW

File: src/rprocessing/applet/BuiltinApplet.java [hunk 1]
```diff
@@ -32,6 +32,8 @@ public BuiltinApplet() {
     Session session =
         new SessionBuilder().withDefaultPackages().setPackageLoader(packageLoader).build();
     this.renjinEngine = new RenjinScriptEngineFactory().getScriptEngine(session);
+    this.renjinEngine.put("key", "0");
+    this.renjinEngine.put("keyCode", 0);
```
Comment (gaocegege) [inline src/rprocessing/applet/BuiltinApplet.java]: Maybe https://github.com/hawkingrei/Processing.R/blob/0edbae2b4e0c1da39cb77d8fd8c54e8fffb27e7d/src/rprocessing/RLangPApplet.java#L133 is a better place to place these `put` statements.

**Comment by hawkingrei** [comment]
fix it

**Comment by gaocegege** [comment]
LGTM

**Comment by jeremydouglass** [comment]
I have been trying to understand whether or not the input function mappings in this commit (keyPressed, mouseClicked, etc.) are implemented, and if so how to use them.

Is there a way to use `keyPressed` or `keyPressed()` in Processing.R based on this commit?

**Comment by gaocegege** [comment]
I am afraid not because of https://github.com/gaocegege/Processing.R/issues/170

**Pull request merged**